code/smart-mobility/smart-mobility.js

```diff
@@ -88,33 +88,33 @@ function bigButton(
 
   if (callback) callback();
 }
-	//>>>HOME TAB BUTTONS
+  //>>>HOME TAB BUTTONS
```

Comment:
this top part is changes that are in prod, but I didnt put them there, must not have made it into github